draft-ietf-anima-brski-prm.md

```diff
@@ -520,127 +540,148 @@ In {{!RFC8995}}, pledges instead need to continuously request enrollment from a
+Akin to the Network Proximity Assertion in BRSKI {{!RFC8995}}, the Agent Proximity Assertion provides pledge proximity evidence to the MASA.
+But additionally, the Agent Proximity Assertion allows the domain registrar to be sure that the PVR supplied by the Registrar-Agent was in fact collected by the Registrar-Agent to which the registrar is connected by utilizing an agent-signed data object.
 
-The provisioning of the Registrar-Agent LDevID certificate is out of scope for this document, but may be done in advance using a separate BRSKI run or by other means like configuration.
-It is recommended to use short lived Registrar-Agent LDevIDs in the range of days or weeks as outlined in {{sec_cons_reg-agt}}.
 
 
-# System Components
 
+# System Components
 
-## Domain Registrar
+## Domain Registrar {#registrar_component}
 
 In BRSKI-PRM, the domain registrar provides the endpoints already specified in {{!RFC8995}} (derived from EST {{!RFC7030}}) where suitable.
-In addition, it MUST provide the endpoints defined in {{registrar_ep_table}} within the BRSKI-defined "/.well-known/brski/" URI path.
-These endpoints accommodate for the signature-wrapped objects used by BRSKI-PRM for the Pledge Enroll-Request (PER) and the provisioning of CA certificates.
+In addition, it MUST provide the endpoints defined in {{registrar_ep_table}} within the BRSKI-defined `/.well-known/brski/` Well-Known URI path.
+These endpoints accommodate for the authenticated self-contained objects used by BRSKI-PRM to provide Pledge Enroll-Request (PER) artifacts and signature-wrapped CA certificates via the Registrar-Agent.
 
 |Endpoint        | Operation                  | Exchange and Artifacts  |
 |:---------------|:---------------------------|:------------------------|
 | requestenroll  | Supply PER to Registrar    | {{per}} |
 |------------------------
-| wrappedcacerts | Request CA Certificates    | {{req_cacerts}} |
+| wrappedcacerts | Obtain CA Certificates    | {{obtain_cacerts}} |
 |===============
-{: #registrar_ep_table title='Additional Well-Known Endpoints on a BRSKI-PRM Registrar' }
+{: #registrar_ep_table title='Additional Well-Known Endpoints on a BRSKI-PRM Registrar'}
+
+The registrar possesses its own EE certificate and corresponding private key for authenticating and signing.
+It MUST use the same certificate/credentials for authentication in the TLS session with a Registrar-Agent and for signing artifacts for that Registrar-Agent and its pledges (see {{rcv_artifact}}).
+If the registrar EE certificate is provided to the Registrar-Agent(s) via configuration, the credential used for signing artifacts for a Registrar-Agent MUST correspond to the registrar EE certificate configured at that Registrar-Agent.
+This ensures that Registrar-Agent and pledge can verify the certficate and signature of the intended domain registrar. TODO("intended" domain registrar is not clear anymore when the RegAgt can change registrar between collecting and providing artifacts)
```

Comment:
What is meant by "intended" domain registrar?
Since it was clarified that there may be multi-registrar installations and the RegAgt may change registrars after obtaining the initial registrar EE cert, there is no one specific intended registrar.

The checks aim at same domain for RegAgt and Registrar. Is this it? Would need language improvement.

Reply:
Agreed, this is bit indirect requirement. The intention was to have registrar-agent and registrar belonging to the same domain. 
Proposal:

If the registrar EE certificate is provided to the Registrar-Agent(s) via configuration, the credential used for signing artifacts for a Registrar-Agent MUST correspond to the registrar EE certificate configured at that Registrar-Agent. 
This ensures that Registrar-Agent can verify the certificate and the signature of artifacts from the domain registrar.

Reply:
Fix by removing restriction as discussed in call.